File: Stack/stack.py
##Stack Implementation using List in python
##create a stack
def createStack():
  stack=[]
  return stack

##check the stack is empty
def isEmpty(stack):
  return len(stack)==0

##Insert element to stack
def push(stack,item):
  stack.append(item)
  print("Pushed Item: "+item)

##Delete element from stack
def pop(stack):
  if(isEmpty(stack)):
    return "Stack is Empty"
  else:
    return stack.pop()

##Main function
stack=createStack()
push(stack,str(11))
push(stack,str(12))
push(stack,str(13))
print("Stack: "+str(stack))
print("Popped Item: "+pop(stack))
print("Stack after Popping: "+str(stack))

File: Stack/Stack.cpp
// Stack implementation in C

#include <iostream>
#include <stdlib.h>

#define MAX 10
using namespace std;
int count = 0;

// Creating a stack
struct stack {
  int items[MAX];
  int top;
};
typedef struct stack st;

void createEmptyStack(st *s) {
  s->top = -1;
}

// Check if the stack is full
int isfull(st *s) {
  if (s->top == MAX - 1)
    return 1;
  else
    return 0;
}

// Check if the stack is empty
int isempty(st *s) {
  if (s->top == -1)
    return 1;
  else
    return 0;
}

// Add elements into stack
void push(st *s, int newitem) {
  if (isfull(s)) {
    cout<<"STACK FULL";
  } else {
    s->top++;
    s->items[s->top] = newitem;
  }
  count++;
}

// Remove element from stack
void pop(st *s) {
  if (isempty(s)) {
    cout<< "\n STACK EMPTY \n";
  } else {
    cout<<"Item popped= "<<s->items[s->top];
    s->top--;
  }
  count--;
  printf("\n");
}

// Print elements of stack
void printStack(st *s) {
  cout<<"Stack: ";
  for (int i = 0; i < count; i++) {
    cout<<endl<< s->items[i];
  }
  cout<<endl;
}

// Driver code
int main() {
  int ch;
  st *s = (st *)malloc(sizeof(st));

  createEmptyStack(s);

  push(s, 1);
  push(s, 2);
  push(s, 3);
  push(s, 4);

  printStack(s);

  pop(s);

  cout<<"\nAfter popping out\n";
  printStack(s);
}